tests/installation/add_serial_console.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Set serial terminal in GRUB for VMware
+# Maintainer: Michal Nowak <mnowak@suse.com>
+
+use strict;
+use base 'y2logsstep';
+use testapi;
+use version_utils 'is_vmware';
+
+sub run {
+    select_console 'install-shell';
+
+    my $device = is_vmware() ? 'sda' : 'vda';
```

Comment:
does not make sense to me . Currently we loading this module **only** for vmware and you stating that problem is vmware specific so there lower chance that it would ever be used for something except vmware so why we calling `is_vmware` inside a module ?